coupling/delwaq/generate.py

```diff
@@ -43,7 +43,7 @@ def generate(toml_path: Path) -> tuple[nx.DiGraph, set[str]]:
     for row in model.node_table().df.itertuples():
         if row.node_type not in ribasim.geometry.edge.SPATIALCONTROLNODETYPES:
             G.add_node(
-                row.node_id,
+                row.node_type + str(row.node_id),
```

Comment:
Not sure exactly what it's used for, but since it is a string representation, let's use the same one we use elsewhere, "Pump #5" instead of "Pump5".

```suggestion
                f"{row.node_type} #{row.node_id}",
```

Reply:
The string representation is only for uniqueness, and is removed soon after. Making it into the visualization equivalent implies that it is used as such later on, so I rather leave it as is.